config/src/sumeragi.rs

```diff
@@ -24,7 +24,7 @@ view! {
     /// `Sumeragi` configuration.
     /// [`struct@Configuration`] provides an ability to define parameters such as `BLOCK_TIME_MS`
     /// and a list of `TRUSTED_PEERS`.
-    #[derive(Debug, Clone, PartialEq, Eq, Deserialize, Serialize, Documented, Combine)]
+    #[derive(Debug, Clone, PartialEq, Eq, Deserialize, Serialize, Documented, Proxy, LoadFromEnv)]
```

Comment:
Why there is no `LoadFromDisk` in such places? It was inside `Combine` before

Reply:
Will be there eventually. It's just that `Configuration` currently has an inherent impl with `from_file` and `load_environment` methods, so it works regardless of derives, and they actually might interfere. I'll make the whole switch when plugging in `Proxy` structs in a future PR.